recipe/build.sh

```diff
@@ -4,6 +4,60 @@
 # -------
 chmod +x configure
 
+if [ $(uname) == Linux ]; then
+    compiler_mkspec=mkspecs/common/g++-base.conf
+    flag_mkspec=mkspecs/linux-g++/qmake.conf
+
+    # The Anaconda gcc7 compiler flags specify -std=c++17 by default, which
+    # activates features that break compilation. Begone!
+    CXXFLAGS=$(echo $CXXFLAGS | sed -E 's@\-std=[^ ]+@@')
+    export CXXFLAGS="$CXXFLAGS -std=c++98"
+
+    # This warning causes a huge amount of spew in the build logs.
+    if [ "$cxx_compiler" = gxx ] ; then
+        CXXFLAGS="$CXXFLAGS -Wno-expansion-to-defined"
+    fi
+
+    export LDFLAGS="$LDFLAGS -Wl,-rpath-link,$PREFIX/lib"
+    export LDFLAGS="$LDFLAGS -Wl,-rpath-link,${BUILD_PREFIX}/${HOST}/sysroot"
+    export CPPFLAGS="$CPPFLAGS -DXK_dead_currency=0xfe6f -DXK_ISO_Level5_Lock=0xfe13"
+    export CPPFLAGS="$CPPFLAGS -DFC_WEIGHT_EXTRABLACK=215 -DFC_WEIGHT_ULTRABLACK=FC_WEIGHT_EXTRABLACK"
+    export CPPFLAGS="$CPPFLAGS -DGLX_GLXEXT_PROTOTYPES"
+else
+    compiler_mkspec=mkspecs/common/clang.conf
+    flag_mkspec=mkspecs/macx-clang/qmake.conf
```

Comment:
On Qt4 archive we don't have this configuration, we only have:
* macx-icc
* macx-llvm
* macx-pbbuilder
* macx-xcode
* macx-xlc

Reply:
We do have `darwin-g++` tho.

Reply:
Sorry I just checked we also have 'macx-g++', 'g++40' and 'g++42'. My eyes are failing me

Reply:
Not a problem, anyway I'm using ``macx-llvm``

Reply:
@marcelotrevisani, Qt 4 probably doesn't compile with the latest Clang, so you should use `g++42` (at least that's what we used in Anaconda).

Reply:
Ok! I will change it! I will just wait the CI linux to finish
Thanks!